src/package_io/sanity_check.star

```diff
@@ -1,5 +1,6 @@
 ROOT_PARAMS = [
     "observability",
+    "challengers",
```

Comment:
shouldn't be something changing in this file, to handle the removal of challenger_params ?

Reply:
ah forgot to remove them, all done now